src/rpc/rawtransaction.cpp

```diff
@@ -366,6 +367,100 @@ static RPCHelpMan getrawtransactionmulti() {
     };
 }
 
+static RPCHelpMan getislocks()
+{
+    return RPCHelpMan{"getislocks",
+        "\nReturns the raw InstantSend lock data for each txids. Returns Null if there is no known IS yet.",
+        {
+            {"txids", RPCArg::Type::ARR, RPCArg::Optional::NO, "The transaction ids (no more than 100)",
+                {
+                    {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, "A transaction hash"},
+                },
+            },
+            {"verbose", RPCArg::Type::NUM, RPCArg::Default{1}, "0 for hex-encoded data, 1 for a json object"},
+        },
+        RPCResult{
+            RPCResult::Type::ARR, "", "Response is an array with the same size as the input txids",
+            {
+                RPCResult{"for verbose = 0 if InstantSend Lock is known for specified txid",
+                     RPCResult::Type::STR, "data", "The serialized, hex-encoded data for 'txid'"
+                },
+                RPCResult{"for verbose = 1 if InstantSend Lock is known for specified txid",
+                    RPCResult::Type::OBJ, "", "",
+                    {
+                        {RPCResult::Type::STR_HEX, "txid", "The transaction id"},
+                        {RPCResult::Type::ARR, "inputs", "The inputs",
+                        {
+                            {RPCResult::Type::OBJ, "", "",
+                                {
+                                    {RPCResult::Type::STR_HEX, "txid", "The transaction id"},
+                                    {RPCResult::Type::NUM, "vout", "The output number"},
+                                },
+                            },
+                        }},
+                        {RPCResult::Type::STR_HEX, "cycleHash", "The Cycle Hash"},
+                        {RPCResult::Type::STR_HEX, "signature", "The InstantSend's BLS signature"},
```

Comment:
Don't think we should mention the algorithm used to generate the signature in the help text.

```suggestion
                        {RPCResult::Type::STR_HEX, "signature", "The InstantSend lock's signature"},
```

Reply:
unified with `getbestchainlock`
```
                {RPCResult::Type::STR_HEX, "signature", "The ChainLock's BLS signature"},
```

Reply:
That should be changed too IMO but change could be out-of-scope for this PR.